include/deal.II/base/vectorization.h

```diff
@@ -69,6 +69,8 @@
 #    undef vector
 #    undef pixel
 #    undef bool
+#  elif defined(__ARM_NEON__)
+#    include <arm_neon.h>
 #  else
 #    include <x86intrin.h>
 #  endif
```

Comment:
```
In file included from /home/ubuntu/jenkins/workspace/dealii-ampere_PR-15923/include/deal.II/base/vectorization.h:75:
In file included from /usr/lib/llvm-15/lib/clang/15.0.7/include/x86intrin.h:15:
/usr/lib/llvm-15/lib/clang/15.0.7/include/immintrin.h:14:2: error: "This header is only meant to be used on x86 and x64 architecture"
#error "This header is only meant to be used on x86 and x64 architecture"
```
Apparently `__ARM_NEON__` is not defined for Ampere which is why it includes the wrong header. Is there a different define for Ampere?